onadata/settings/kc_environ.py

```diff
@@ -144,6 +144,15 @@
     if not os.path.isdir(EMAIL_FILE_PATH):
         os.mkdir(EMAIL_FILE_PATH)
 
+# Local time zone for this installation. Choices can be found here:
+# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
+# although not all choices may be available on all operating systems.
+# On Unix systems, a value of None will cause Django to use the same
+# timezone as the operating system.
+# If running in a Windows environment this must be set to the same as your
+# system time zone.
+TIME_ZONE = os.environ.get('TIME_ZONE', 'America/New_York')
```

Comment:
This note is everywhere in kobocat next to TIME_ZONE setting:

https://github.com/qedsoftware/kobocat/blob/5ea087f388384d594e1a1287e11241499db41b8c/onadata/settings/common.py#L45

https://github.com/qedsoftware/kobocat/blob/5ea087f388384d594e1a1287e11241499db41b8c/onadata/settings/production_example.py#L44

I have just copy pasted it, so that it is consistent across project.